#two Ai motors    (0~300 degree  or 0~1023 resolution)

#uint8[2] AI_id
uint16[2] AI_degree

#one servo motor   (0~180 degree)

uint16 Servo_degree
